inc/inventorycomputerlib.class.php

```diff
@@ -1342,28 +1342,11 @@ function updateComputer($a_computerinventory, $computers_id, $no_history, $setdy
                      && isset($arraydb['serial'])
                      && $arrayslower['serial'] == $arraydb['serial']
                   ) {
-                     $update = false;
-                     if ($arraydb['capacity'] == 0
-                              && $arrayslower['capacity'] > 0) {
-                        $input = [
-                           'id'       => $keydb,
-                           'capacity' => $arrayslower['capacity']
-                        ];
-                        $update = true;
-                     }
-
-                     if ($arraydb['voltage'] == 0
-                              && $arrayslower['voltage'] > 0) {
-                        $input = [
-                           'id'        => $keydb,
-                           'voltage'   => $arrayslower['voltage']
-                        ];
-                        $update = true;
-                     }
```

Comment:
Voltage belongs to DeviceBattery, not Item_DeviceBattery. It is useless to handle it here as the column does not exists.